ui-tests/src/test/java/io/syndesis/qe/utils/ExcludeFromSelectorReports.java

```diff
@@ -0,0 +1,12 @@
+package io.syndesis.qe.utils;
+
+import java.lang.annotation.ElementType;
+import java.lang.annotation.Retention;
+import java.lang.annotation.RetentionPolicy;
+import java.lang.annotation.Target;
+
+@Retention(RetentionPolicy.RUNTIME)
+@Target(ElementType.METHOD)
```

Comment:
Maybe we could consider targeting whole classes as well, that would simplify the apicurito and atlasmap steps above. Not necessary though

Reply:
I also thought about that, but apicurito steps have some syndesis related methods... and also if some methods get added in the future there's no clear indication that some bytecode manipulation is performed per method... and also it'd be more work :D